File: documentation/user/en/get-started/example/filter-order-entities.java
return evita.queryCatalog(
	"evita",
	session -> {
		return session.queryListOfSealedEntities(
			query(
				collection("Brand"),
				filterBy(
					attributeStartsWith("name", "A")
				),
				orderBy(
					attributeNatural("name", OrderDirection.ASC)
				),
				require(entityFetchAll())
			)
		);
	}
);

File: documentation/user/en/get-started/example/filter-order-entities.cs
return evita.QueryCatalog(
	"evita",
	session => {
		return session.QueryListOfSealedEntities(
			Query(
				Collection("Brand"),
				FilterBy(
					AttributeStartsWith("name", "A")
				),
				OrderBy(
					AttributeNatural("name", OrderDirection.Asc)
				),
				Require(EntityFetchAll())
			)
		);
	}
);